components/deepgram/deepgram.app.mjs

```diff
@@ -0,0 +1,178 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
+export default {
+  type: "app",
+  app: "deepgram",
+  propDefinitions: {
+    projectId: {
+      type: "string",
+      label: "Project",
+      description: "Identifier of a project",
+      async options() {
+        const { projects } = await this.listProjects();
+        return projects?.map(({
+          name, project_id: id,
+        }) => ({
+          label: name,
+          value: id,
+        })) || [];
+      },
+    },
+    balanceId: {
+      type: "string",
+      label: "Balance",
+      description: "Identifier of a balance",
+      async options({ projectId }) {
+        const { balances } = await this.listBalances({
+          projectId,
+        });
+        return balances?.map(({ balance_id: id }) => id) || [];
+      },
+    },
+    requestId: {
+      type: "string",
+      label: "Request",
+      description: "Identifier of a request",
+      async options({
+        projectId, page,
+      }) {
+        const { requests } = await this.listRequests({
+          projectId,
+          params: {
+            page,
+          },
+        });
+        return requests?.map(({
+          path, request_id: id,
+        }) => ({
+          label: path,
+          value: id,
+        })) || [];
+      },
+    },
+    language: {
+      type: "string",
+      label: "Language",
+      description: "BCP-47 language tag that hints at the primary spoken language",
```

Comment:
Can you provide a link with docs about `BCP-47` or an exemple of how to fill the prop like e.g. `en-US`

Reply:
or options / async options